code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -1,5 +1,5 @@
 /obj/item/weapon/gun/projectile/automatic/pistol
-	name = "\improper FK-69 pistol"
+	name = "stechkin pistol"

```

Comment:
I swear to god I'm just going to go back to "syndicate pistol" and have a description about a stolen design of forgotten origin.